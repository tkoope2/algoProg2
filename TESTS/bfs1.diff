8,19c8,19
< @ a1 : dist=2 ; npaths=0
< @ a2 : dist=3 ; npaths=0
< @ a3 : dist=4 ; npaths=0
< @ a4 : dist=5 ; npaths=0
< @ b1 : dist=1 ; npaths=0
< @ b2 : dist=2 ; npaths=0
< @ b3 : dist=3 ; npaths=0
< @ b4 : dist=4 ; npaths=0
< @ c1 : dist=0 ; npaths=0
< @ c2 : dist=1 ; npaths=0
< @ c3 : dist=2 ; npaths=0
< @ c4 : dist=3 ; npaths=0
---
> @ a1 : dist=2 ; npaths=1
> @ a2 : dist=3 ; npaths=3
> @ a3 : dist=4 ; npaths=6
> @ a4 : dist=5 ; npaths=10
> @ b1 : dist=1 ; npaths=1
> @ b2 : dist=2 ; npaths=2
> @ b3 : dist=3 ; npaths=3
> @ b4 : dist=4 ; npaths=4
> @ c1 : dist=0 ; npaths=1
> @ c2 : dist=1 ; npaths=1
> @ c3 : dist=2 ; npaths=1
> @ c4 : dist=3 ; npaths=1
