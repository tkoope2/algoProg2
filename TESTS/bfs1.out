@1++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
@2
@3 COMMAND-LINE:
@4
@5    ./bfs rings.g c1 
@6
@7++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
@ a1 : dist=2 ; npaths=0
@ a2 : dist=3 ; npaths=0
@ a3 : dist=4 ; npaths=0
@ a4 : dist=5 ; npaths=0
@ b1 : dist=1 ; npaths=0
@ b2 : dist=2 ; npaths=0
@ b3 : dist=3 ; npaths=0
@ b4 : dist=4 ; npaths=0
@ c1 : dist=0 ; npaths=0
@ c2 : dist=1 ; npaths=0
@ c3 : dist=2 ; npaths=0
@ c4 : dist=3 ; npaths=0
